code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -347,6 +351,7 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 	throw_range = 3
 	turf_type = /turf/open/floor/clockwork
 	novariants = FALSE
+	grind_results = list("iron" = 0.5, "teslium" = 1.5)
```

Comment:
But brass is made of copper (and zinc). There's no iron in it.

Reply:
"Brass sheets" are actually just replicant alloy fused into the sturdy form that looks like brass but is strong enough to build with (normal brass probably wouldn't make good solid walls.)